Add ssl support for epsonworkforce.markdown

cociweb opened the pull request:
## Proposed change
Add ssl support for Epson Workforce integration



## Type of change


- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ]  Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes https://github.com/home-assistant/core/issues/84070

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

c0ffeeca7 committed:
Assist: add more examples of built-in sentences (#30054)

c0ffeeca7 committed:
SmartThings: fix link (#30059)

joncar committed:
Update link to programming software. (#30061)

c0ffeeca7 committed:
Moehlenhoff: fix dead link (#30086)

DroneDaddy committed:
Updated yolink.markdown to include Outdoor Motion Detector Sensor support (#30074)

I have tested the YoLink Outdoor Motion Detector Sensor (YS7805) and confirmed an automation worked within Home Assistant 2023.11.3.

rccoleman committed:
Fix cover state checks (#30083)

Two `value_template` templates were comparing the state of a cover entity to "on"/"off" in one place, and "open"/"closed" in another.  The latter are the correct state values.

FletcherAU committed:
Add note about google cred activation delay fixes #29430 (#30088)

koestler committed:
add go-iotdevie into the list of third-party tools supporting homeassistant discovery (#30023)

Co-authored-by: Jan Bouwhuis <jbouwh@users.noreply.github.com>

AJediIAm committed:
Update google.markdown (#30090)

When following the steps, the text "and click 'Go to credentials'" confused me since I was not offered such a button. The next step brought me to where I needed to go. 
The text "and click 'Go to credentials'" is not needed.

Shulyaka committed:
openai_conversation: Update documentation to reflect the newer models (#30096)

* openai_conversation: Update documentation to reflect the newer language models

* Missing period.

mhaowork committed:
Update mqtt.markdown to fix typo (#30100)

There's one too many right bracket causing the json parsor to fail.

Error log:

2023-12-01 23:45:07.964 WARNING (MainThread) [homeassistant.components.mqtt.discovery] Unable to parse JSON irrigation: '{"name": "Irrigation", "command_topic": "homeassistant/switch/irrigation/set", "state_topic": "homeassistant/switch/irrigation/state", "unique_id": "irr01ad", "device": {"identifiers": ["garden01ad"], "name": "Garden" }}}'

FletcherAU committed:
Clarify Slack member ID retrieval (#30091)

* Document member ID retrieval in house

* Normalise usage of member vs user

* Change instructions to ordered list

* Update source/_integrations/slack.markdown

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

* Update slack.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

FletcherAU committed:
Clarify how Slack uses messages and blocks (#30089)

* Document Slack block limit per message

* Explain where blocks and message fields will be used by Slack

* Update example to clarify block/message usage

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

FletcherAU committed:
Move information on CT clamps to a template (#30107)

FletcherAU committed:
Fix anchor link to point to correct section of page (#30108)

FletcherAU committed:
Reword SSDP note and admin access explainer (#30111)

FletcherAU committed:
Link to suitable notification integrations (#30112)

FletcherAU committed:
Replace Google password auth instructions with expanded app password info (#30106)

* Replace Google password auth instructions with expanded app password info

* tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

nwithan8 committed:
Add missing forecast_template to configuration variables (#30119)

FletcherAU committed:
Move tts_service from example to configuration (#30113)

* Move tts_service from example to configuration

* Implement feedback by the author of this PR

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

starkillerOG committed:
Clearify independend Reolink push notification switch (#30123)

fyhertz committed:
Enhance Ubus Integration Documentation & Refine ACL Permissions (#30121)

* Enhance Ubus Integration Documentation & Refine ACL Permissions

The current documentation does not contain enough information to start using the ubus integration, and the current ACL config file gives more permission than is actually needed.

This change explains how to add a system user `hass` and documents the configuration of `/etc/config/rpcd` configuration file. Those steps are missing from the current configuration.

Credits for those valuable steps go to: https://github.com/kvj/hass_openwrt.

Further more, the current ACLs config file gives more permission than is actually needed to make the integration work.

* Tiny style tweaks

* Update source/_integrations/ubus.markdown

* Update source/_integrations/ubus.markdown

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

c0ffeeca7 committed:
Linux VirtuaBox installation: update (#30101)

c0ffeeca7 committed:
Assist: add troubleshooting on weather queries (#30073)

FletcherAU committed:
Abode intro tense and perspective correction (#30133)

FletcherAU committed:
Capitalise list items in Acmeda (#30134)

FletcherAU committed:
Fix word order in Actiontec intro (#30135)

GuySie committed:
Add Matter Geneva blog (#30082)

* Add Matter Geneva blog

* Rename 2023-11-30-nabu-cas-at-the-matter-member-meeting to 2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown

* Update source/_posts/2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

* Rename 2023-11-30-nabu-cas-at-the-matter-member-meeting.markdown to 2023-11-30-nabu-casa-at-the-matter-member-meeting.markdown

* Update and rename 2023-11-30-nabu-casa-at-the-matter-member-meeting.markdown to 2023-12-04-nabu-casa-at-the-matter-member-meeting.markdown

---------

Co-authored-by: Paulus Schoutsen <paulus@home-assistant.io>

FletcherAU committed:
Fix capitalisation in Agent DVR (#30142)

FletcherAU committed:
Minor error correction and readability improvements for Namecheap's DNS (#30145)

FletcherAU committed:
Correct spelling errors and minor formatting issues in z-wave (#30143)

FletcherAU committed:
Restructure Adax instrurctions (#30136)

FletcherAU committed:
Slightly adjust word order of cups and remove duplicate note (#30147)

c0ffeeca7 committed:
Assist troubleshooting: add step how to use alias (#30155)

cociweb committed:
Add ssl support for epsonworkforce.markdown

c0ffeeca7 commented:
Hi @cociweb :wave: 

Thank you for contributing.

As this documentation change depends on a parent PR being merged in the core code base, this docs PR should be pointing to the `next` branch.
1. Rebase and point it to the `next` branch.
2. Under **Link to parent pull request**, add a link to your core PR. 
3. Please fix the issues mentioned by the text linter. You can see what they are by selecting the **Details**.

Marking this PR as **Draft** until these points have been resolved. Once you are ready, mark it as **Ready for review**.

Commit:
Update thethingsnetwork.markdown (#30140)

* Update thethingsnetwork.markdown

Suggest MQTT for TTNv3

* Tiny tweak

---------

Co-authored-by: c0ffeeca7 <38767475+c0ffeeca7@users.noreply.github.com>

tomlut committed:
Update sensor.rest.markdown (#30154)

c0ffeeca7 committed:
Installation on Windows, using VMware Workstation: update procedure (#30161)

* Installation on Windows: VMware update procedure

- update procedure to work with latest versioin of VMware workstation 17 Player
- Procedure is mostly based on / Credits to SimplySmartHouse on https://community.home-assistant.io/t/installing-home-assistant-on-vmware-player-17/570428
- Procedure was reproduced on a Windows 10 system, using a HAOS v 11.1 image.
- Implements feedback from #30128

* Fix typo, add troubleshooting step

* Fix typo

* Implement review feedback
